src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/prisonperson/PrisonPersonReconService.kt

```diff
@@ -0,0 +1,64 @@
+package uk.gov.justice.digital.hmpps.prisonertonomisupdate.prisonperson
+
+import com.microsoft.applicationinsights.TelemetryClient
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.async
+import kotlinx.coroutines.withContext
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.prisonertonomisupdate.config.trackEvent
+import uk.gov.justice.digital.hmpps.prisonertonomisupdate.nomissync.model.PrisonPersonReconciliationResponse
+import uk.gov.justice.digital.hmpps.prisonertonomisupdate.prisonperson.model.PhysicalAttributesDto
+import uk.gov.justice.digital.hmpps.prisonertonomisupdate.services.awaitBoth
+
+@Service
+class PrisonPersonReconService(
+  private val dpsApi: PrisonPersonDpsApiService,
+  private val nomisApi: PrisonPersonNomisApiService,
+  private val telemetryClient: TelemetryClient,
+) {
+
+  suspend fun checkPrisoner(offenderNo: String) = runCatching {
```

Comment:
Added support for a single prisoner but it's not plumbed in yet - eventually this will be called for all active prisoners.